.github/workflows/_reusable_app_release.yml

```diff
@@ -5,7 +5,6 @@ on:
         required: true
         type: string
       is_cloud_build:
-        required: true
```

Comment:
Is it then possible to remove the last change again?
https://github.com/wireapp/wire-ios/pull/1296

Reply:
good point, this change was on release branch already right ? I would keep this one like this, and let's see if there's no more issues before removing it.

Reply:
both, release branch and develop